trace_test.go

```diff
@@ -63,6 +64,49 @@ func TestBundling(t *testing.T) {
 	}
 }
 
+func TestBundling_ConcurrentExports(t *testing.T) {
+	workers := 2
+	delay := 2 * time.Second
+	exporter := newTraceExporterWithClient(Options{
+		ProjectID:            "fakeProjectID",
+		BundleCountThreshold: 10,
+		BundleDelayThreshold: delay,
+		NumberOfWorkers:      workers,
+	}, nil)
+
+	wg := sync.WaitGroup{}
+	waitCh := make(chan struct{})
+	wg.Add(workers)
+
+	exporter.uploadFn = func(spans []*tracepb.Span) {
```

Comment:
Consider counting the number of exported spans and comparing it against the expected number.

Reply:
Done!